.evergreen/config.yml

```diff
@@ -456,14 +446,9 @@ axes:
         batchtime: 10080  # 7 days
 
       # NOTE: reenable when these are actually running 7.2, or we release a 7.4 rpm
-      # - id: rhel72-zseries-test
-      #   display_name: "RHEL 7.2 (zSeries)"
-      #   run_on: rhel72-zseries-test
-      #   batchtime: 10080  # 7 days
-
-      - id: suse12-zseries-test
-        display_name: "SUSE 12 (zSeries)"
-        run_on: suse12-zseries-test
+      - id: rhel72-zseries-test
```

Comment:
Should this be `rhel67-zseries-test`, which is what the [drivers ticket ](https://jira.mongodb.org/browse/DRIVERS-469) says?

Reply:
yep! nice catch thanks

Reply:
hm looks like we can't actually build over there due to gcc version, let's let this PR sit until someone on dev prod can update on that

Reply:
@kvwalker it was determined we should stick with rhel72, reverted